sdk/src/main/java/io/opentelemetry/sdk/trace/export/ExporterConstants.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.export;
+
+import java.util.concurrent.TimeUnit;
+
+/** Provides constants for all exporter default configuration values. */
+public class ExporterConstants {
+
+  private ExporterConstants() {}
+
+  /** Common default host name. */
+  public static final String DEFAULT_HOST_NAME = "unknown";
+  /** Common default service name. */
+  public static final String DEFAULT_SERVICE_NAME = DEFAULT_HOST_NAME;
+
+  public static final long DEFAULT_DEADLINE_MS = TimeUnit.SECONDS.toMillis(1); // 1 second
+
+  /** Default endpoint for otlp. */
+  public static final String OTLP_DEFAULT_ENDPOINT = "localhost:55680";
```

Comment:
I don't think it's a good idea to centralize these constants that are only used by the specific exporters. Keep the knowledge close to the place that needs it.